src/API/Reports/PerformanceIndicators/Controller.php

```diff
@@ -177,17 +184,43 @@ public function get_analytics_report_data() {
 		}
 	}
 
+	/**
+	 * Get active Jetpack modules.
+	 *
+	 * @return array List of active Jetpack module slugs.
+	 */
+	public function get_active_jetpack_modules() {
+		if ( is_null( $this->active_jetpack_modules ) ) {
+			if ( class_exists( '\Jetpack' ) && method_exists( '\Jetpack', 'get_active_modules' ) ) {
+				$this->active_jetpack_modules = \Jetpack::get_active_modules();
```

Comment:
I think this can be set to `null` here if someone filters `jetpack_active_modules` and returns null, so it would get called repeatedly, but that's super edge case, I suppose.

Reply:
Handled in 26523280.